django/conf/locale/tr/formats.py

```diff
@@ -14,7 +14,8 @@
 # The *_INPUT_FORMATS strings use the Python strftime format syntax,
 # see https://docs.python.org/library/datetime.html#strftime-strptime-behavior
 DATE_INPUT_FORMATS = [
-    '%d/%m/%Y', '%d/%m/%y',     # '25/10/2006', '25/10/06'
+    '%d/%m/%Y',  # '25/10/2006'
+    '%d/%m/%y',  # '25/10/06'
     '%y-%m-%d',                 # '06-10-25'
```

Comment:
Space of this comment looks a little off?

Reply:
Yes, but `black` will change it. It didn't want to change unaffected lines.